pkgs/os-specific/linux/kernel/common-config.nix

```
@@ -868,11 +868,12 @@ let
     };
 
     zram = {
-      ZRAM           = module;
-      ZRAM_WRITEBACK = option yes;
-      ZSWAP          = option yes;
-      ZPOOL          = yes;
-      ZBUD           = option yes;
+      ZRAM               = module;
+      ZRAM_WRITEBACK     = option yes;
+      ZRAM_DEF_COMP_ZSTD = whenAtLeast "5.11" yes;
```

```suggestion
      ZRAM_DEF_COMP_ZSTD = whenAtLeast "5.11" yes;
      ZSWAP_COMPRESSOR_DEFAULT_ZSTD = whenAtLeast "5.19" yes;
```
If you select `zstd` as the default compressor for Zram, perhaps you should do the same for Zswap (for the sake of consistency).